src/Analyzers/Core/Analyzers/Formatting/FormatterHelper.cs

```diff
@@ -30,10 +30,10 @@ internal static IEnumerable<AbstractFormattingRule> GetDefaultFormattingRules(IS
     /// <param name="cancellationToken">An optional cancellation token.</param>
     /// <returns>The formatted tree's root node.</returns>
     public static SyntaxNode Format(SyntaxNode node, ISyntaxFormatting syntaxFormattingService, SyntaxFormattingOptions options, CancellationToken cancellationToken)
-        => Format(node, SpecializedCollections.SingletonEnumerable(node.FullSpan), syntaxFormattingService, options, rules: null, cancellationToken: cancellationToken);
+        => Format(node, [node.FullSpan], syntaxFormattingService, options, rules: null, cancellationToken: cancellationToken);
```

Comment:
> [node.FullSpan] [](http://example.com/codeflow?start=23&length=16)

Does this allocate more now that it uses collection exprs? SharpLab seemed to indicate that there is an array allocated in the underlying class, which would seem suboptimal in the single element case.

Reply:
I should have read the description. Out of curioisty, do we have any say in the sharplab versions, as it would be nice to validate this behavior before switching to it.

Reply:
Sharplab will eventually update.  This should be optimal thanks to the work done by Alrz here: https://github.com/dotnet/roslyn/pull/71147